usaspending_api/api_contracts/contracts/v2/autocomplete/recipient.md

```diff
@@ -0,0 +1,61 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# Recipient Autocomplete [/api/v2/autocomplete/recipient/]
+
+This endpoint is used by the Recipient autocomplete filter on the Advanced Search page.
+
+## POST
+
+This route sends a request to the backend to retrieve recipients matching the specified search text. The endpoint will search for `search_text` in the `recipient_name` and `uei` fields. The `recipient_levels` value can also be included to further filter results, but is not required.
+
++ Request (application/json)
+    + Schema
+
+            {
+                "$schema": "http://json-schema.org/draft-04/schema#",
+                "type": "object"
+            }
+
+    + Attributes (object)
+        + `limit` (optional, number)
+            + Default: 10
+        + `search_text` (required, string)
+        + `recipient_levels` (optional, array[string])
+    + Body
+
+            {
+                "search_text": "Holdings"
+            }
+
++ Response 200 (application/json)
```

Comment:
I would add a messages field here. Even though it won't be used...right now, it may be used in the future. I ran into this issue recently where an existing endpoint we deprecated didn't have a messages field so I had to change the api contract to include it. Please see, https://github.com/fedspendingtransparency/usaspending-api/pull/3995/files#diff-e35f25abf9df4847c3431eb10b6a969760b808de98bde1b7c96410f00d2d7284, for an example.

Reply:
Okay so if I understand this correctly, just adding the message field here and nothing else? Don't have to change the API itself right?

Reply:
If you add a new field to the api contract you will have to populate it right? It's ok if it's an empty list. See my link for an example